src/types/types.ts

```diff
@@ -191,13 +191,22 @@ export interface Asset
   parentExternalId?: CogniteExternalId;
 }
 
-export interface AggregateResponse {
+export type AggregateResponse = CountAggregateResponse;
```

Comment:
I don't think we should introduce aliases.
can't we just use AggregateResponse directly?